packages/web-components/src/components/ic-back-to-top/ic-back-to-top.tsx

```diff
@@ -110,8 +110,8 @@ export class BackToTop {
     this.setFooterVisible(entries[0].isIntersecting);
   };
 
-  private findTargetEl = (target: string): Element => {
-    let targetElement: Element = null;
+  private findTargetEl = (target: string): Element | null => {
```

Comment:
Later, create a simple type for `Element | null` in a util file and import it where it is used?

Reply:
Thank you for your suggestion - I might leave it for this PR if that is ok, but something we will definitely consider in future, especially if we start using the `Element | null` type a bit more often